webapp/channels/src/components/user_settings/security/user_settings_security.tsx

```diff
@@ -495,7 +495,7 @@ export class SecurityTab extends React.PureComponent<Props, State> {
             describe = (
                 <FormattedMessage
                     id='user.settings.security.loginOffice365'
-                    defaultMessage='Login done through Office 365'
+                    defaultMessage='Login done through Entra IDS'
```

Comment:
```suggestion
                    defaultMessage='Login done through Entra ID'
```

Reply:
Looks like a typo in this line with "IDS" instead of "ID", if I understand the changes correctly?

Reply:
Yes, not sure how that got in there. Good catch! Fixed.